openshift-hack/images/Dockerfile.openshift

```diff
@@ -0,0 +1,14 @@
+
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.9 as builder
+
+WORKDIR /build
+COPY . .
+RUN make cloud-controller-manager
+
+FROM registry.ci.openshift.org/ocp/4.9:base
+
+LABEL description="Alibaba Cloud Controller Manager"
+
+COPY --from=builder /build/alibaba-cloud-ccm /bin/alibaba-cloud-ccm
+
+ENTRYPOINT [ "/bin/alibaba-cloud-ccm" ]
```

Comment:
Elsewhere, we have the binary called `platform-cloud-controller-manager`, would be good to be consistent. Is this something that's in the upstream? Can we rename it in the dockerfile if so?

Reply:
sure, will be done!